public/video-ui/src/components/FormFields/ScribeEditor.js

```diff
@@ -24,6 +24,10 @@ export default class ScribeEditorField extends React.Component {
   };
 
   getWords = text => {
+    if (!text) {
+      return 0;
```

Comment:
`updateWordCount` also does `this.getWords(text).length` which won't work if `getWords` returns `0`